common/environments/python-sdk-v2/context/Dockerfile

```diff
@@ -15,3 +15,5 @@ RUN conda env create -p $CONDA_PREFIX -f conda_dependencies.yaml -q && \
     rm conda_dependencies.yaml && \
     conda run -p $CONDA_PREFIX pip cache purge && \
     conda clean -a -y
+    
+RUN pip install applicationinsights
```

Comment:
Please consider moving this to conda_dependencies.yaml instead.

Reply:
Done